docs/DesignDocs/CustomAttestation.md

```diff
@@ -0,0 +1,420 @@
+___
+
+Custom Attestation Data Formats for Open Enclave
+=====
+
+This design document proposes a new attestation framework and set of APIs that
+enable developers to use custom formats for their attestation evidence and
+collateral.
+
```

Comment:
Evidence and collateral are new terminologies to the OE SDK audience, it would be a good idea to define them. 
Also explains attester,v erifier, relying party, and asserter and so on.

It would be even better you could provide link to RATS doc for naming referring.

Reply:
I'll add a terminology section